github-dark.css

```diff
@@ -1276,35 +1276,22 @@
     outline-color: /*[[base-color]]*/ #4f8cc9 !important;
   }
   input.focus[type="text"], #adv_code_search .focus.search-page-label,
-  /* below selector has a ton of weight. Better to override it with
-   * class selectors that unset box-shadow and rules here. Testable
-   * on the repo tag editor */
-  input[type="text"]:focus:not(.js-color-editor-input):not(.tree-finder-input):not(.js-site-search-focus):not(.tag-input-inner),
   .focused .drag-and-drop, #adv_code_search .search-page-label:focus,
-  input.focus[type="password"], input.form-control[type="password"]:focus,
-  input.focus[type="email"], input.form-control[type="email"]:focus,
-  input.focus[type="number"], input.form-control[type="number"]:focus,
-  input.focus[type="tel"], input[type="tel"]:focus, input.focus[type="url"],
-  input.form-control[type="url"]:focus, textarea.focus, textarea:focus,
+  .form-control:focus, .form-select:focus,
   #q:focus ~ button, input.color-editor-input:focus, #ghd-settings select:focus,
   #ghd-settings input:focus, .intgrs-lstng-item:hover,
   .ajax-pagination-btn:focus, .form-control.focus, .form-select.focus,
   .form-select:focus, input[type="checkbox"]:focus, .upload-enabled.focused,
-  .project-column:focus, .project-card:focus {
+  .project-column:focus, .project-card:focus,
+  .octotree_github_sidebar input[type="text"]:focus,
+  .octotree_github_sidebar textarea:focus {
     border-color: /*[[base-color]]*/ #4f8cc9 !important;
     box-shadow: inset 0 1px 2px rgba(0, 0, 0, .075), 0 0 2px /*[[base-color]]*/ #4f8cc9 !important;
     outline-color: /*[[base-color]]*/ #4f8cc9 !important;
   }
   .dragover textarea, .dragover .drag-and-drop {
     box-shadow: 0 0 1px 1px /*[[base-color]]*/ #4f8cc9 !important;
   }
-  /* inputs */
-  select, input:not(.btn):not(.btn-link):not(.btn-danger), textarea {
-    -moz-appearance: none !important;
```

Comment:
Looks like this is still neccesary to style `input[type=checkbox]` on Firefox. First checkbox below does not have it:

<img width="109" alt="screenshot 2018-12-10 at 19 40 22" src="https://user-images.githubusercontent.com/115237/49753376-7d84c980-fcb3-11e8-8f52-e2b234c0f567.png">

Reply:
Is that on the PR page? If so I'm running FF 65.0b2 (64-bit) and not seeing the issue.

Actually, with GHD disabled I don't see checkboxes on that page at all.

Reply:
Interesting, in Chrome those checkboxes show, but in Edge & FF they don't. In FF there's a space for them, and in Edge there isn't.

Reply:
That's from the PR page, yes. This may be related to me running Firefox 57. Oh, and the same goes for `input[type="radio"]`.

Reply:
Actually, it appears Mozilla now supports our existing `-webkit-appearance`, but not in version 57. Can you just add `-moz-appearance` in the place we have `-webkit-appearance` for checkboxes/radios.

Reply:
Looks like FF was caching part of GHD even when it was disabled, that's why my checkboxes weren't showing up.